src/coreclr/jit/gentree.cpp

```diff
@@ -26944,6 +26961,28 @@ void ReturnTypeDesc::InitializeStructReturnType(Compiler*                comp,
 #endif
 }
 
+#ifdef SWIFT_SUPPORT
+//---------------------------------------------------------------------------------------
+// InitializeSwiftReturnRegs:
+//   Initialize the Return Type Descriptor for a method that returns with the
+//   Swift calling convention.
+//
+// Parameters:
+//   comp   - Compiler instance
+//   clsHnd - Struct type being returned
+//
+void ReturnTypeDesc::InitializeSwiftReturnRegs(Compiler* comp, CORINFO_CLASS_HANDLE clsHnd)
+{
+    const CORINFO_SWIFT_LOWERING* lowering = comp->GetSwiftLowering(clsHnd);
+    assert(!lowering->byReference);
+    assert(lowering->numLoweredElements <= MAX_RET_REG_COUNT);
```

Comment:
`MAX_RET_REG_COUNT` is already 4 on ARM64, but it might be helpful to the reader to `static_assert` that here.

Reply:
I added a static assert.